app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,40 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
+
+        //encode it as a zip file
+        var content = zipUtil.encode(false, this._io, temp_dir);
+        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
+
+        //delete the temp dir
+        function rmDirRecursive(io, dir){
```

Comment:
I was saying that:
> I realized that synchronous `rmDirRecursive` is part of the io API already, see: https://github.com/graphicore/ufoJS/blob/master/lib/tools/io/_base.js#L214
>
> You can do `this._io.rmDirRecursive(false, temp_dir);`

But `io.rmDirRecursive` is dependant on `io.stat` and that is not defined everywhere :-/

So forget this :-)

There is already a series of issues for that: #22 #23 #24